src/Nethermind/Nethermind.JsonRpc/Modules/Proof/ProofRpcModule.cs

```diff
@@ -143,6 +144,23 @@ public ResultWrapper<TransactionWithProof> proof_getTransactionByHash(Keccak txH
 
             return ResultWrapper<TransactionWithProof>.Success(txWithProof);
         }
+        
+        private int SumOfPreviousLogIndexes(Block block, TxReceipt txReceipt)
```

Comment:
Can we share this logic across all modules?

Reply:
Share the logic between the parity one too